lib/runtime/src/ops/element_binary.h

```diff
@@ -2,15 +2,19 @@
 #define _FLEXFLOW_ELEMENT_BINARY_H
 
 #include "op-attrs/ops/element_binary.h"
-#include "op_task_invocation.h"
 #include "sim_environment.h"
 
 namespace FlexFlow {
 
+OpTaskSignature init_signature();
```

Comment:
```suggestion
template <>
OpTaskSignature init_signature<ELEMENTBINARY_INIT_TASK_ID>();
```

Same for `fwd_signature()` and `bwd_signature()`